dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/services/FileVerificationService.kt

```diff
@@ -0,0 +1,49 @@
+package org.dataland.documentmanager.services
+
+import org.apache.pdfbox.Loader
+import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import org.springframework.web.multipart.MultipartFile
+import java.io.IOException
+
+/**
+ * A service for performing basic sanity checks on files uploaded by user
+ */
+@Component
+class FileVerificationService {
```

Comment:
Should we revert that renaming commit then?

Reply:
However you like.   I am also ok with the current name, because before the verificator does its job, you are not 100% sure that you have an actual pdf at hand.   So at this point, it is indeed a file verificator.

On the other hand, it is looking for pdfs, so pdf verificator would also be ok (if the name shall contain the thing it is looking for, rather than the thing it is actually scanning).

So you tell me.  I am 50 50.

Reply:
Sorry for the effort done. But I'd prefer the original version :/

Reply:
Ok no problem. Will start doing it now

Reply:
Done